cmd/cli/install_test.go

```diff
@@ -1620,3 +977,77 @@ func createDeploymentSpec(namespace, meshName string) *v1.Deployment {
 		},
 	}
 }
+
+func getDefaultInstallCmd(writer *bytes.Buffer) installCmd {
```

Comment:
Do these values need to stay in sync with the defaults in install.go? If so, can we define and make use of the defaults there and reference them here in the tests so we don't have to manually keep them in sync?

Reply:
Done